server/src/metadata/field-metadata/field-metadata.auto-resolver-opts.ts

```diff
@@ -0,0 +1,44 @@
+import { SortDirection } from '@ptc-org/nestjs-query-core';
+import {
+  AutoResolverOpts,
+  PagingStrategies,
+  ReadResolverOpts,
+} from '@ptc-org/nestjs-query-graphql';
+
+import { JwtAuthGuard } from 'src/guards/jwt.auth.guard';
+
+import { FieldMetadata } from './field-metadata.entity';
+
+import { FieldMetadataService } from './services/field-metadata.service';
+import { CreateFieldInput } from './dtos/create-field.input';
+import { UpdateFieldInput } from './dtos/update-field.input';
+
+export const fieldMetadataAutoResolverOpts: AutoResolverOpts<
```

Comment:
I've extracted this config from the module to make it more readable, I thought some part of it could be shared but still not sure if it's necessary

Reply:
@Weiko I'm not sure if it's a good option to put them in another file